src/ukrainian-vareniki.md

```diff
@@ -0,0 +1,34 @@
+# Ukrainian Vareniki
+
+(Vareniki/Dumplings) Vareniki are popular all over the world, but aboard they have different names. 
+
+- ⏲️ Prep: 30 mins
+- 🍳 Cook time: 25 mins  
+- 🍽️ Servings: 45
+
+## Ingredients 
```

Comment:
```suggestion
## Ingredients
```

Reply:
btw, this is why the check was failing. The current script only checks that the line is exactly `## Ingredients` and doesn't allow for whitespace on the end.

The new script at #462 Has more detailed checks for whitespace but it hasn't been merged